CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 - Use `Logger.debug` to print warning about closing keep-alive connections.
 
+## Fixed
```

Comment:
This isn't part of `0.15.4`. will need to go above `0.15.4` section. just start one called master if empty.